src/transport/SecureSessionMgr.cpp

```diff
@@ -175,17 +180,23 @@ CHIP_ERROR SecureSessionMgr::SendPreparedMessage(SessionHandle session, const En
 
         destination = &state->GetPeerAddress();
 
-        ChipLogProgress(Inet, "Sending %s msg %p to 0x" ChipLogFormatX64 " at utc time: %" PRId64 " msec", "encrypted",
-                        &preparedMessage, ChipLogValueX64(state->GetPeerNodeId()), System::Clock::GetMonotonicMilliseconds());
+        ChipLogProgress(Inet,
+                        "Sending %s msg %p with MessageCounter:" ChipLogFormatMessageCounter " to 0x" ChipLogFormatX64
+                        " at utc time: %" PRId64 " msec",
+                        "encrypted", &preparedMessage, preparedMessage.GetMessageCounter(), ChipLogValueX64(state->GetPeerNodeId()),
+                        System::Clock::GetMonotonicMilliseconds());
     }
     else
     {
         auto unauthenticated = session.GetUnauthenticatedSession();
         mUnauthenticatedSessions.MarkSessionActive(unauthenticated.Get());
         destination = &unauthenticated->GetPeerAddress();
 
-        ChipLogProgress(Inet, "Sending %s msg %p to 0x" ChipLogFormatX64 " at utc time: %" PRId64 " msec", "plaintext",
-                        &preparedMessage, ChipLogValueX64(kUndefinedNodeId), System::Clock::GetMonotonicMilliseconds());
+        ChipLogProgress(Inet,
+                        "Sending %s msg %p with MessageCounter:" ChipLogFormatMessageCounter " to 0x" ChipLogFormatX64
+                        " at utc time: %" PRId64 " msec",
+                        "plaintext", &preparedMessage, preparedMessage.GetMessageCounter(), ChipLogValueX64(kUndefinedNodeId),
+                        System::Clock::GetMonotonicMilliseconds());
```

Comment:
Monotonic milliseconds is not UTC time, but time since boot.

Reply:
Preexisting, but will fix.